docs/usage.md

```diff
@@ -106,19 +106,24 @@ The parameter `--map-base-url` maps external schema references to local folders.
 When using our docker image that we regularly update, you don't need to install Node.js or Npm, even though the generator is written with it since the Docker image has the generator installed.
 
 Install [Docker](https://docs.docker.com/get-docker/) first, then use docker to pull and run the image using the following command:
+``` 
+and run the image using the following command :
```

Comment:
```suggestion
```

this test was there already, a line above

also it was breaking in preview
![Screenshot 2023-07-13 at 17 46 06](https://github.com/asyncapi/generator/assets/6995927/73c2ca33-1db1-47ee-93c0-a19f14c578b4)